client/modules/team/components/MyTeam.js.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { PropTypes } from 'react';
+import Base from "../../../global/components/BaseComponent.js.jsx";
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+// import actions
+import { singleActions } from '../actions';
+import { listActions } from '../actions';
+import { listActions as playerListActions } from '../../player/actions';
+import { singleActions as playerSingleActions } from '../../player/actions';
+import PlayerListItem from '../../player/components/PlayerListItem.js.jsx';
+import { listActions as userListActions } from '../../user/actions';
+import { singleActions as userSingleActions } from '../../user/actions';
```

Comment:
you are importing a TON of things here. userListActions and the respective userListActions.fetchList called below are never actually used, and slow the page down. I'd remove anything you don't use in the render function.